fmn/consumer/backend.py

```diff
@@ -14,21 +14,19 @@
 import smtplib
 import time
 
+from fedmsg_meta_fedora_infrastructure import fasshim
+from pika.adapters import twisted_connection
+from twisted.internet import defer, reactor, protocol, task
 import pika
 import fedmsg
 import fedmsg.meta
 import fedmsg_meta_fedora_infrastructure
 
-from pika.adapters import twisted_connection
```

Comment:
This seems like a PEP-8 move - I recommend doing this in a separate commit to keep this one cleaner and focused.